test/templates/config.ru

```diff
@@ -0,0 +1,70 @@
+$:.push File.dirname(__FILE__)
+require "fake_middleware"
```

Comment:
I think you can even define the `FakeMiddleware` class here and save the `LOAD_PATH` juggling.